Dockerfile.rhel7

```diff
@@ -1,8 +1,20 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.12 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.12 AS builder
+
+RUN dnf install -y git make go
+
 WORKDIR /go/src/github.com/openshift/must-gather
 COPY . .
 ENV GO_PACKAGE github.com/openshift/must-gather
 
 FROM registry.ci.openshift.org/ocp/4.12:cli
 COPY --from=builder /go/src/github.com/openshift/must-gather/collection-scripts/* /usr/bin/
-RUN yum install --setopt=tsflags=nodocs -y jq && yum clean all && rm -rf /var/cache/yum/*
+RUN yum install --setopt=tsflags=nodocs -y jq 
+RUN yum install -y pciutils util-linux hostname rsync tar ethtool
+RUN yum clean all && rm -rf /var/cache/yum/*
+
+ARG NODE_GATHER_MANIFESTS_DIR=performance-profile-node-gather/
+
+COPY ${NODE_GATHER_MANIFESTS_DIR} /etc/performance-profile-node-gather
+
+# the binary follows the must-gather convention of helper tools called gather_$SOMETHING
+COPY build/_output/bin/gather-sysinfo /usr/bin/gather_sysinfo
```

Comment:
I don't quite understand the lines above, what are they doing?

Reply:
After generate a binary with make we need to copy it to the image